tests/integration/docker/exec.bats

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+#Based on docker commands
+
+setup() {
+	load common
+	cleanDockerPs
+	runtimeDocker
+}
+
+@test "modifying a container with exec" {
+	# This problem is related with https://github.com/01org/cc-oci-runtime/issues/18
+	skip "exec is still not working"
+	$DOCKER_EXE run --name containertest -p 6001:6011 -d ubuntu bash -c "sleep 300"
+	$DOCKER_EXE ps -a | grep "Up"
+	$DOCKER_EXE exec -d containertest bash -c "echo 'hello world' > file"
+	$DOCKER_EXE exec containertest bash -c "cat /file"
```

Comment:
Could you grep for the expected result? that will add the check we can send data to stdout.